packages/SwingSet/src/kernel/vatManager/manager-local.js

```diff
@@ -141,6 +142,7 @@ export function makeLocalVatManagerFactory(tools) {
       endowments,
       inescapableTransforms,
       inescapableGlobalLexicals,
+      inescapableGlobalProperties,
```

Comment:
This needs to be added to all the managers, not just `manager-local.js`

Reply:
I was starting to write a response to the effect that I completely don't understand what this means.  I took it to mean that I should translate my changes to `manager-local.js` into corresponding changes to the other `manager-xxxx.js` files, but when I looked at these files I found they have no analog to the code being modified here (e.g., call to `makeLiveSlots` and use of the return value from that to set up the call to `importBundle`).  This aligned with my original impression  that `manager-local` is some base implementation that's used by the others, as we do some of the other vat worker implementation, hence the other workers would acquire the new logic that way.  But some grepping through the code reveals that actually the corresponding code I need to change is in the various `supervervisor-xxxx.js` files, so I think I now see where to make the adjustments.  But this confusion points to what seems to me like a modularity issue with how our code is spread among the various files.

Reply:
Yes, that's right. "manager" here means "code that runs in the kernel process", and "supervisor" means "code that runs in the worker process/thread". `manager-local.js` is special because both pieces run in the same process, so there is no `supervisor-local.js`. For all the other worker types, there is both `manager-X.js` and `supervisor-X.js`.

The new tools are needed by the worker side, because that's where the vat bundle gets evaluated. So we need to change the three `supervisor-*.js` files to include these changes, in addition the code that would have been named `supervisor-local.js` if that weren't combined with `manager-local.js`.

(I did a bunch of refactoring last month to move a lot of common code into `manager-helper.js` and `supervisor-helper.js`, so most of what's left in `supervisor-*.js` is actually worker-type -specific now)